enterprise-ws-spec/src/main/asciidoc/ClientProgrammingModel.adoc

```diff
@@ -99,14 +93,14 @@ client developer. The WSDL agnostic methods must be used if the client
 developer has a partial WSDL definition that only contains only the
```

Comment:
".... only contains only ...." typo we can fix or does this make sense to native speaker?